frontend/taipy-gui/src/components/Taipy/Field.tsx

```diff
@@ -75,6 +84,14 @@ const Field = (props: TaipyFieldProps) => {
                 <span className={className} id={id} style={style}>
```

Comment:
could you change `className={className}` to `className={'className getSuffixedClassNames(className, '-'+mode)'}` for pre, markdown, MathJax & span ?
at least will be able to test that taipy-text-latex

Reply:
lost in translation ?